base_classes/NXdata.nxdl.xml

```diff
@@ -85,124 +85,170 @@
 
 		.. code-block::
 
 			data:NXdata
 			  @signal = "data"
-			  @axes = ["x", "y"]  --> the order does NOT matter
+			  @axes = ["x", "y"]
 			  @x_indices = [0, 1]
 			  @y_indices = [0, 1]
 			  data: float[10,20]
 			  x: float[10,20]     --> coordinates along both dimensions
 			  y: float[10,20]     --> coordinates along both dimensions
 
-		In this example each data point ``data[i,j]`` has axis coordinates ``[x[i,j], y[i,j]]``.
-
-		`Dimensions`
-
-		The data dimensions annotated by an :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;` field are defined by the
-		:ref:`AXISNAME_indices &lt;/NXdata@AXISNAME_indices-attribute&gt;` attribute. When this attribute is missing,
-		the position(s) of the :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;` string in the
-		:ref:`axes &lt;/NXdata@axes-attribute&gt;` attribute are used.
+		In this example each data point ``data[i,j]`` has axis coordinates ``[x[i,j], y[i,j]]`` and when
+		plotting, `x` is used along the first data dimension by default and `y` along the second data dimension.
+		Since `x` and `y` span both dimensions, a reader could choose to use `y` for the first dimension
+		and `x` for the second as an alternative to the default. A writer could also choose to not specify
+		any default by specifying `@axes = [".", "."]` and leave the decision up the the reader.
 
-		When all :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;` fields are one-dimensional, and none of the data dimensions
-		have more than one axis, the :ref:`AXISNAME_indices &lt;/NXdata@AXISNAME_indices-attribute&gt;` attributes
-		are often omitted. If one of the data dimensions has no :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;` field,
-		the string “.” can be used in the corresponding index of the axes list.
+		Note that omitting `@x_indices` would result in `@x_indices = [0]` as derived from the position of `"x"` in `@axes`.
+		This would be invalid since the shape `[10,20]` of `x` is not equal to the shape `[10]` of the spanned data dimensions.
 
-		An example of this would be
+		Omitting indices for multi-dimensional axes can only be done by repeating the :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;`
+		name in all positions of the :ref:`axes &lt;/NXdata@axes-attribute&gt;` attribute which they span.
 
 		.. code-block::
 
 			data:NXdata
 			  @signal = "data"
-			  @axes = ["x", ".", "z"]  --> the order matters
-			  data: float[10,20,30]
-			  x: float[10]             --> coordinates along the first dimension
-			  z: float[30]             --> coordinates along the third dimension
+			  @axes = ["x", "x"]
+			  data: float[10,20]
+			  x: float[10,20]     --> coordinates along both dimensions
+
+		Note that since ``NXdata`` does not describe how the data is to be plotted or even the dimensionality of the plot,
+		a reader would probably treat this particular example as a 1D signal and plot it as such.
+
+		`3. Dimensions spanned by more than one axis`
```

Comment:
How does this differ from the title in L144: `2. Multi-dimensional axes`

Reply:
one axis spanning multiple dimensions vs. one dimension covered by more that one axis

Reply:
I added this to make it more clear:

"The first dimension is covered by three axes: `x`, `y` and `x_set`. The second dimension is also covered by three axes: `x`, `y` and `y_set`."

Reply:
In addition I will put the second example first because indeed, `2. Multi-dimensional axes` has both Multi-dimensional axes and axes spanned by more than one dimension.